### PR TITLE
Parser update on size 0

### PR DESCRIPTION
**Pull request by jjsjann123**
Disables fusion of empty tensor (numel == 0), which causes scheduler asserts with our latest scheduler update.

### CODE REVIEW

**Review by kevinstephano — APPROVED** [review]
LGTM

File: torch/csrc/jit/codegen/cuda/parser.cpp [hunk 1]
```diff
@@ -69,6 +69,16 @@ bool is_reduction_non_compatible_tensor(
   return is_zero_dim_tensor(tensor_type) || is_zero_sized_tensor(tensor_type);
 }
 
+bool is_input_non_size_zero_tensor(const Node* node) {
```
Comment (rdspring1) [inline torch/csrc/jit/codegen/cuda/parser.cpp]: Nit: isInputNonSizeZeroTensor

**Review by rdspring1 — APPROVED** [review]
LGTM

File: torch/csrc/jit/codegen/cuda/parser.cpp (hunk 1 above)
Comment (jjsjann123) [reply torch/csrc/jit/codegen/cuda/parser.cpp]: Ha good catch!

**Pull request merged**